src/stories/packages/components/ButtonOutline.stories.tsx

```diff
@@ -0,0 +1,66 @@
+import type { Meta, StoryObj } from '@storybook/react';
+import { ButtonOutline } from '@terra-ui-packages/components';
+import { ThemeProvider } from '@terra-ui-packages/components';
+
+import React from 'react';
+
+import { defaultBrand } from 'src/libs/brands';
+
+// More on how to set up stories at: https://storybook.js.org/docs/writing-stories#default-export
+const meta = {
+  title: 'Packages/Components/ButtonOutline',
+  component: ButtonOutline,
+  parameters: {
+    // Optional parameter to center the component in the Canvas. More info: https://storybook.js.org/docs/configure/story-layout
```

Comment:
All of these comments came from an example story that was installed as part of injecting Storybook. I left them for now to give pointers to folks, but would likely remove them before merging.